controllers/profiles/reconciler_dev.go

```diff
@@ -324,11 +358,108 @@ func mountDevConfigMapsMutateVisitor(flowDefCM, propsCM *v1.ConfigMap) mutateVis
 			deployment.Spec.Template.Spec.Containers[0].VolumeMounts = make([]v1.VolumeMount, 0)
 			deployment.Spec.Template.Spec.Containers[0].VolumeMounts = volumeMounts
 
+			for keycm, cm := range externalResourceConfigMaps {
+				deployment.Spec.Template.Spec.Volumes = append(deployment.Spec.Template.Spec.Volumes,
+					v1.Volume{
+						Name: cm.Name,
+						VolumeSource: v1.VolumeSource{
+							ConfigMap: &v1.ConfigMapVolumeSource{
+								LocalObjectReference: v1.LocalObjectReference{Name: cm.Name},
+							},
+						},
+					})
+				deployment.Spec.Template.Spec.Containers[0].VolumeMounts = append(deployment.Spec.Template.Spec.Containers[0].VolumeMounts,
+					v1.VolumeMount{
+						Name:      cm.Name,
+						ReadOnly:  true,
+						MountPath: getMountPath(keycm),
+					})
+			}
 			return nil
 		}
 	}
 }
 
+func getMountPath(name string) string {
+	var mountPath = quarkusDevConfigMountPath
+	if strings.HasSuffix(name, extResCamelSuffix) {
+		return configMountPathMap[CAMEL]
+	}
+	if strings.HasSuffix(name, extResGenericSuffix) {
+		return configMountPathMap[GENERIC]
+	}
+	if strings.HasSuffix(name, extResAsyncAPISuffix) {
+		return configMountPathMap[ASYNCAPI]
+	}
+	if strings.HasSuffix(name, extResOpenAPISuffix) {
+		return configMountPathMap[OPENAPI]
+	}
+	return mountPath
+}
+
+func loadExternalResourcesConfigMaps(support *stateSupport, workflow *operatorapi.KogitoServerlessWorkflow) map[string]v1.ConfigMap {
+	externalConfigMaps := make(map[string]v1.ConfigMap, 0)
+	for k, val := range workflow.Labels {
+		resource, found := cutPrefix(k, metadata.Domain+"/")
+		if found {
+			cm, err := getGenericConfigMap(val, workflow.Namespace, support)
+			if err != nil {
+				log.Errorf(err, "ConfigMap not found %s", val)
+			} else {
+				externalConfigMaps[resource] = cm
+			}
+		}
+	}
+	return externalConfigMaps
+}
+
+func cutPrefix(s string, prefix string) (string, bool) {
```

Comment:
Can't we get rid of this?

Reply:
@ricardozanini  I think not, 
the ` sw.kogito.kie.org/resource-` was to identify our labels for the external resources, if we get rid of this
a user can write a label like
 ` acme.org/resource-camel` and will be treated as a valid for us because is one of te valid prefix even if isn't our label

Reply:
Yeah, I'm not saying not to do the logic, but figure out a better way of doing it. This `cutPrefix` method seems a workaround in this context to me.

Reply:
@ricardozanini done, with a string split